### ISSUE TITLE
Creating new world gets stuck at 0%

### ISSUE THREAD

**Issue by LatvianModder** [FTBTeam/FTB-Omnia#36]
* **Modpack Version**: 1.2.0

* **Issue**: ![](https://i.latvian.dev/pc/2020-04-16_04.51.29.png)

* **Is it Repeatable?**: Yes

Log didn't contain anything interesting. Have fun with this one.

These are the last lines:
```
[16Apr2020 04:51:18.541] [Server thread/INFO] [placebo/]: Registered 0 additional loot tables.
[16Apr2020 04:51:24.689] [Server thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 2983 advancements
[16Apr2020 04:51:27.615] [Server thread/ERROR] [net.blay09.mods.kleeslabs.KleeSlabs/]: Slab wood_slab_0 could not be found.
[16Apr2020 04:51:27.615] [Server thread/ERROR] [net.blay09.mods.kleeslabs.KleeSlabs/]: Slab wood_slab_1 could not be found.
[16Apr2020 04:51:27.615] [Server thread/ERROR] [net.blay09.mods.kleeslabs.KleeSlabs/]: Slab other_slab could not be found.
[16Apr2020 04:51:27.621] [Server thread/INFO] [BonsaiTrees2/]: Loaded 24 bonsai types and 6 soil types
[16Apr2020 04:51:27.627] [Server thread/INFO] [BonsaiTrees2/]: Updated soil compatibility
[16Apr2020 04:52:41.792] [Server thread/INFO] [net.minecraftforge.common.DimensionManager/DIMS]: Registered dimension the_bumblezone:bumblezone of type the_bumblezone:ultraamplified and id 8
[16Apr2020 04:52:48.799] [Server thread/WARN] [net.minecraft.world.server.ServerWorld/]: Unable to find spawn biome
```

**Comment by vertex101** [comment]
i run the modpack with 6gb of ram and this works fine for me so it might be a ram issue

https://streamable.com/6ww2ge

---UPDATE---

after doing some more research into this found out the minimum the modpack can do is 4gb of ram anything under it will not load up a SP world and causes the very high CPU usage

**Comment by Sunekaer** [comment]
Issue is that 3GB is not enough.
With 3GB ram you so close to the limit, that the garbage collector is running ALL the time.
the world gen is not stuck at 0% its just hitting ram and cpu limit as the garbage collector is running ALL the time. you can see in the log file that there happens some here and there.

tldr 
Assign more ram
